yt_dlp/extractor/rdsit.py

```diff
@@ -0,0 +1,79 @@
+import re
+
+from .common import InfoExtractor
+
+
+class RDSITIE(InfoExtractor):
+    IE_NAME = 'rds.it'
+    _VALID_URL = r'https?://(?:www\.)?rds\.(?:it|radio)'
+    _TESTS = [{
+        'url': 'https://rds.it',
+        'info_dict': {
+            'id': 'rds',
+            'ext': 'mp4',
+            'title': r're:^RDS',
```

Comment:
```suggestion
            'title': r're:RDS',
```

See https://discord.com/channels/807245652072857610/1112613156934668338/1293369656098160702. `re.match` is used here when running tests(which matches from the start of the string already)